packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.js

```diff
@@ -101,7 +101,6 @@ const MoonstoneDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	addAll({
 		cancel: 461,
 		nonModal: [
-			461,
```

Comment:
I haver some concerns here.  Currently `nonModal` is only used in Spotlight, but that doesn't mean that it might not be used by apps or elsewhere in Moonstone in the future.  Is there any harm in leaving it here?

Reply:
The inconsistency between webOS and non-webOS is a "harm" imo. If we were to leave it, I'd want to consider it deprecated and remove it in 3.0.

I think your concern is valid but given the complete absence of any documentation of `nonModal`, it seems like a low risk to me.

Reply:
Which inconsistency are you referring to?  The handling of escape in Spotlight if `moonstone` is not included?

Reply:
That "back" on webOS is nonModal but "back" on PC isn't.

Reply:
Is the code in `spotlight` added below enough to fix that inconsistency or is there a further issue?

Reply:
Or is this just a comment on the weird key handling of the magic remote?

Reply:
My rationale for leaving the key entry in moonstone is that it is a non-modal key on the magic remote.  This might be important at some point.  I think adding the additional fix to spotlight is fine, too.

Reply:
I think our approach for these keys should be revisited but not in the context of this ticket. I'll revert this change and we can revisit later.

In short, our definition of "non-modal" at inception was "key codes that don't cause pointer mode to be disabled." In hindsight, I think we'd have been better served by saying "key names that don't cause pointer mode to be disabled" in which case we would have had a list that included "cancel" and would have worked cross-platform.